### PR TITLE
izs-campus env

### PR DESCRIPTION
**Pull request by benmaidel**

### CODE REVIEW

**Comment by jabbenseth** [comment]
fails due to missing "object_groups.yaml" for izs-* environments

```
-- run_tests.py: execute commands
  /usr/bin/cmake -E make_directory /root/catkin_ws/build/cob_default_env_config/test_results/cob_default_env_config
  /opt/ros/indigo/share/roslaunch/cmake/../scripts/roslaunch-check -o /root/catkin_ws/build/cob_default_env_config/test_results/cob_default_env_config/roslaunch-check___ROBOT_ENV_ipa-kitchen..xml /root/catkin_ws/src/cob_environments/cob_default_env_config/. ROBOT_ENV=ipa-kitchen
...writing test results to /root/catkin_ws/build/cob_default_env_config/test_results/cob_default_env_config/roslaunch-check___ROBOT_ENV_izs-carpark-top..xml
FAILURE:
[/root/catkin_ws/src/cob_environments/cob_default_env_config/./upload_object_locations.launch]:
	error loading <rosparam> tag: 
	file does not exist [/root/catkin_ws/src/cob_environments/cob_default_env_config/envs/izs-carpark-top/object_groups.yaml]
XML is <rosparam command="load" file="$(find cob_default_env_config)/envs/$(arg robot_env)/object_groups.yaml" ns="object_groups"/>
wrote test file to [/root/catkin_ws/build/cob_default_env_config/test_results/cob_default_env_config/roslaunch-check___ROBOT_ENV_izs-carpark-top..xml]
...writing test results to /root/catkin_ws/build/cob_default_env_config/test_results/cob_default_env_config/roslaunch-check___robot_env_izs-campus..xml
FAILURE:
[/root/catkin_ws/src/cob_environments/cob_default_env_config/./upload_object_locations.launch]:
	error loading <rosparam> tag: 
	file does not exist [/root/catkin_ws/src/cob_environments/cob_default_env_config/envs/izs-campus/object_groups.yaml]
XML is <rosparam command="load" file="$(find cob_default_env_config)/envs/$(arg robot_env)/object_groups.yaml" ns="object_groups"/>
wrote test file to [/root/catkin_ws/build/cob_default_env_config/test_results/cob_default_env_config/roslaunch-check___robot_env_izs-campus..xml]
checking *.launch in directory /root/catkin_ws/src/cob_environments/cob_default_env_config/.
...writing test results to /root/catkin_ws/build/cob_default_env_config/test_results/cob_default_env_config/roslaunch-check___robot_env_izs-carpark-top..xml
FAILURE:
[/root/catkin_ws/src/cob_environments/cob_default_env_config/./upload_object_locations.launch]:
	error loading <rosparam> tag: 
	file does not exist [/root/catkin_ws/src/cob_environments/cob_default_env_config/envs/izs-carpark-top/object_groups.yaml]
XML is <rosparam command="load" file="$(find cob_default_env_config)/envs/$(arg robot_env)/object_groups.yaml" ns="object_groups"/>
wrote test file to [/root/catkin_ws/build/cob_default_env_config/test_results/cob_default_env_config/roslaunch-check___robot_env_izs-carpark-top..xml]
checking /root/catkin_ws/src/cob_environments/cob_default_env_config/./upload_navigation_goals.launch
checking /root/catkin_ws/src/cob_environments/cob_default_env_config/./upload_object_locations.launch
...writing test results to /root/catkin_ws/build/cob_default_env_config/test_results/cob_default_env_config/roslaunch-check___ROBOT_ENV_izs-campus..xml
wrote test file to [/root/catkin_ws/build/cob_default_env_config/test_results/cob_default_env_config/roslaunch-check___ROBOT_ENV_izs-campus..xml]
FAILURE:
[/root/catkin_ws/src/cob_environments/cob_default_env_config/./upload_object_locations.launch]:
	error loading <rosparam> tag: 
	file does not exist [/root/catkin_ws/src/cob_environments/cob_default_env_config/envs/izs-campus/object_groups.yaml]
```

**Comment by jabbenseth** [comment]
still missing for izs-carpark-top (shame on me for not enabling the tests when adding a new env)

**Review by jabbenseth — DISMISSED** [review]

**Review by ipa-fez — DISMISSED** [review]

**Comment by ipa-fez** [comment]
@ipa-jba ping

**Comment by ipa-fke** [comment]
Still WIP? @ipa-bnm

**Review by stefandoerr — APPROVED** [review]

File: cob_default_env_config/envs/izs-campus/line_maps/lines_top_deck.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (jabbenseth) [inline cob_default_env_config/envs/izs-campus/line_maps/lines_top_deck.yaml]: remove file? and cob_default_env_config/envs/izs-campus/line_maps/ground_marking.yaml

**Review by jabbenseth — APPROVED** [review]

**Review by jabbenseth — CHANGES_REQUESTED** [review]
clumsy me approving while requesting changes

**Comment by benmaidel** [comment]
ping @ipa-jba

**Comment by stefandoerr** [comment]
ping @ipa-jba  could you review before you're gone for next week

**Review by jabbenseth — APPROVED** [review]

**Pull request merged**